lua/autorun/client/cl_devtools.lua

```diff
@@ -0,0 +1,133 @@
+local ranOnce = !ranOnce
+if (!ranOnce) then return end -- just to prevent edge cases
+ranOnce = true
+
+local isDev = isDev
+local refreshTime = 2
+local rootFolder = rootFolder or nil
+local fileTimes = fileTimes or {}
```

Comment:
I'm pretty sure what this is trying to do is read the global fileTimes and store it into a different local variable. If the file should be re-runnable without overriding the value of filetimes, make it a global